datatypes/src/collections/feature_collection_builder.rs

```diff
@@ -97,6 +98,15 @@ where
             _collection_type: PhantomData,
         }
     }
+
+    pub fn batch_builder(
+        self,
+        output_type: VectorDataType, // TODO: derive from CollectionType
```

Comment:
Yeah, I guess it makes no sense as a parameter.

Reply:
yes get it from the Geometry trait. And/or use a new trait for this.

Reply:
3b07edb26b881591ae01ad68e7d7132d3d436635